charts/sourcegraph/tests/checksum_test.yaml

```diff
@@ -0,0 +1,60 @@
+---
+suite: checksumTest
+templates:
+- frontend/sourcegraph-frontend.Deployment.yaml
+tests:
+- it: should update checksum when pgsql secret is modified
+  set:
+    pgsql:
+      auth:
+        existingSecret: "my-secret"
+  asserts:
+      - equal:
+          path: spec.template.metadata.annotations.[checksum/auth]
+          value: "b960ae6db8c72ca9248278a63593de67f1ddfb9805f82eded969ffde767406db"
```

Comment:
I think this would be a good use case of https://github.com/quintush/helm-unittest/blob/master/README.md#snapshot-testing

Reply:
Thanks, updated. Snapshots don't make sense for the initial feature development (I need to actually establish that the annotation changes first), but it does help now.